packages/components/src/filter-picker/README.md

```diff
@@ -9,36 +9,55 @@ Modify a url query parameter via a dropdown selection of configurable options. T
 import { FilterPicker } from '@woocommerce/components';
 
 const renderFilterPicker = () => {
-	const filters = [
-		{ label: 'Breakfast', value: 'breakfast' },
-		{
-			label: 'Lunch',
-			value: 'lunch',
-			subFilters: [
-				{ label: 'Meat', value: 'meat', path: [ 'lunch' ] },
-				{ label: 'Vegan', value: 'vegan', path: [ 'lunch' ] },
-				{
-					label: 'Pescatarian',
-					value: 'fish',
-					path: [ 'lunch' ],
-					subFilters: [
-						{ label: 'Snapper', value: 'snapper', path: [ 'lunch', 'fish' ] },
-						{ label: 'Cod', value: 'cod', path: [ 'lunch', 'fish' ] },
-						// Specify a custom component to render (Work in Progress)
-						{
-							label: 'Other',
-							value: 'other_fish',
-							path: [ 'lunch', 'fish' ],
-							component: 'OtherFish',
-						},
-					],
-				},
-			],
-		},
-		{ label: 'Dinner', value: 'dinner' },
-	];
+	const config = {
+    label: 'Meal',
+    staticParams: [],
+    param: 'meal',
+    showFilters: function showFilters() {
+        return true;
+    },
+    filters: [{
```

Comment:
Nitpicky, but would you mind updating this to the WooCommerce Admin code styling?

Usually this repo creates a new line for multi-lines arrays.  An example of this can be seen in the [filter picker story](https://github.com/woocommerce/woocommerce-admin/blob/main/packages/components/src/filter-picker/stories/index.js).

Unfortunately, the linter won't pick this up in .md files.

Reply:
@joshuatf I have made the changes, kindly review now. Thanks!!